test/Spec/BotPlutusInterface/Contract.hs

```diff
@@ -898,7 +903,16 @@ assertCommandHistory state =
 assertCommandEqual :: String -> Text -> Text -> Assertion
 assertCommandEqual err expected actual
   | commandEqual expected actual = return ()
-  | otherwise = assertFailure $ err ++ "\nExpected:\n" ++ show expected ++ "\nGot:\n" ++ show actual
+  | otherwise =
+    assertFailure $
+      err ++ "\n" ++ prettyPrintDiff expected actual
+
+prettyPrintDiff :: Text -> Text -> String
+prettyPrintDiff expected actual =
+  "\nExpected:\n"
+    ++ Text.unpack (Diff.above (Wrap 80) FullContext (Text.replace "\n" " " expected) actual)
```

Comment:
why are "\n" being removed only from the expected?

Reply:
The `actual` doesn't include any new lines. Of course we could add it, but it wouldn't have any effect.

Reply:
I'm assuming that this function will only be used to compare CLI commands.